site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -0,0 +1,62 @@
+describe('Test Rainbow Grading', () => {
```

Comment:
```suggestion
import { skipOn } from '@cypress/skip-test';

skipOn(Cypress.env('run_area') === 'CI', () => {
    describe('Test Rainbow Grading', () => {
```

Cypress tests are failing I believe due to the lack of submissions in the CI, so to get this merged before they are added, add this to skip on CI, and we will fix the tests if they break when we add submissions.